qa/integration/specs/monitoring_api_spec.rb

```diff
@@ -51,6 +52,31 @@
     end
   end
 
+  context "verify global event counters" do
```

Comment:
Can we also add a test for `cloned` events? Presumably this should also deal with #8753?

Reply:
Yes, added with commit c2c9bcf5bacd9a2cab2e734d4636f5f7f4ba0323